optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/AbstractConstraintStreamTest.java

```diff
@@ -56,6 +60,11 @@ protected void assumeBavet() {
                 constraintStreamImplType == ConstraintStreamImplType.BAVET);
     }
 
+    protected void assumeDrools() {
+        Assume.assumeTrue("This functionality is not yet supported in Bavet constraint streams.",
+                constraintStreamImplType == ConstraintStreamImplType.DROOLS);
```

Comment:
This PR marks the moment where Drools-based CS learned something that Bavet can not do yet. :-)